rush-plugins/rush-migrate-subspace-plugin/src/cli.ts

```diff
@@ -0,0 +1,36 @@
+import { Command } from 'commander';
+import inquirer from 'inquirer';
+import inquirerSearchList from 'inquirer-search-list';
+import { syncVersions } from './syncVersions';
+import { generateReport } from './generateReport';
+import { migrateProject } from './migrateProject';
+import Console from './providers/console';
+import { interactMenu } from './interactMenu';
+
+inquirer.registerPrompt('search-list', inquirerSearchList);
+
+const program: Command = new Command();
+
+program
+  .option('--analyze', 'to generate all version mismatches for a single project')
```

Comment:
According to the code, it seems to me that `--analyze` was renamed to `--report`?